source/common/network/filter_manager_impl.cc

```diff
@@ -88,6 +88,19 @@ void FilterManagerImpl::onRead() {
   onContinueReading(nullptr, connection_);
 }
 
+bool FilterManagerImpl::startUpstreamSecureTransport() {
+  for (auto& filter : upstream_filters_) {
+    if (!(filter)->filter_) {
```

Comment:
nit: compare with nullptr and this if can be combined with the one below:

`if (filter->filter_ != nullptr && filter->filter_->startUpstreamSecureTransport())`

Reply:
Done. Thanks.